core/src/main/java/org/apache/accumulo/core/metrics/MetricsUtil.java

```diff
@@ -76,6 +77,8 @@ private static void initializeMetrics(boolean enabled, boolean jvmMetricsEnabled
 
       if (address != null) {
         tags.add(Tag.of("Address", address.toString()));
+      } else {
+        throw new NullArgumentException(); 
```

Comment:
Why not IllegalArgumentException with a description - no need to add a new dependency for a standard exception.